test/system/projects.test.ts

```diff
@@ -0,0 +1,34 @@
+import { requestsManager } from 'snyk-request-manager';
+import { updateProject } from '../../src/lib';
+
+const SNYK_API_TEST = process.env.SNYK_API_TEST as string;
+const PROJECT_ID = process.env.TEST_PROJECT_ID as string;
+const ORG_ID = process.env.TEST_ORG_ID as string;
+
+jest.unmock('snyk-request-manager');
+jest.requireActual('snyk-request-manager');
+
+
+describe('UpdateProject', () => {
```

Comment:
If you want an e2e test for it, it would be best to randomly generate the branch each time so you know it was actually updated. Right now once the branch is changed once we will keep updating it to the same branch

Reply:
I changed the branch twice in the test, so it shouldn't be a problem.